src/include/index/bwtree.h

```diff
@@ -108,25 +101,7 @@ namespace bwtree {
 #endif
 
 // This needs to be always here
-static void dummy(const char*, ...) {}
-
-#ifdef BWTREE_DEBUG
-
-#define bwt_printf(fmt, ...)                              \
-  do {                                                    \
-    if(print_flag == false) break;                        \
-    fprintf(stderr, "%-24s(%8lX): " fmt, __FUNCTION__, std::hash<std::thread::id>()(std::this_thread::get_id()), ##__VA_ARGS__); \
-    fflush(stdout);                                       \
-  } while (0);
-
-#else
-
-#define bwt_printf(fmt, ...)   \
-  do {                         \
-    dummy(fmt, ##__VA_ARGS__); \
-  } while (0);
-
-#endif
+static void dummy(const char *, ...) {}
```

Comment:
Should we just get rid of this now?

Reply:
I think these should be deleted because these are used nowhere.

Reply:
I've deleted its dummy function.